### PR TITLE
Add !default to sb-admin-2 SCSS variables

### PR DESCRIPTION
**Pull request by goat-code-cgn**
Added `!default` to all sb-admin-2 SCSS variables so they can be overwritten and customized when importing the SCSS in an own project.

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -50,27 +50,30 @@ $box-shadow: 0 0.15rem 1.75rem 0 rgba($gray-900, .15) !default;
...
-$card-cap-bg: $gray-100;
-$card-border-color: $border-color;
+$card-cap-bg: $gray-100 !default;
+$card-border-color: $border-color !default;
 
 // Adjust column spacing for symmetry
-$spacer: 1rem;
-$grid-gutter-width: $spacer * 1.5;
+$spacer: 1rem !default;
+$grid-gutter-width: $spacer * 1.5 !default;
```
Comment (dpordomingo) [inline scss/_variables.scss]: Why was this closed @davidtmiller?
This PR is really needed, especially this one `$grid-gutter-width` in order to let the user to configure the bootstrap gutter.

**Comment by dpordomingo** [comment]
This is really needed, more details at https://github.com/BlackrockDigital/startbootstrap-sb-admin-2/issues/320

**Comment by dpordomingo** [comment]
superseded by #321

File: scss/_variables.scss (hunk 1 above)
Comment (davidtmiller) [reply scss/_variables.scss]: Sorry about that, I must have closed this one by mistake. We're getting ready to do a little bit of an overhaul to SB Admin 2 but in the meantime this is helpful information.

So, the Bootstrap variables use `!default` as fallbacks essentially, making them used unless they are previously defined.

I wanted to hard-override the Bootstrap variables by removing `!default` from the SB Admin 2 variables - I am wondering what use case you are specifically referring to and how you would be using theses.

**Pull request merged**